inc/saiwred.h

```diff
@@ -92,9 +106,6 @@ typedef enum _sai_wred_attr_t
     /** 0 ~ 15 [sai_uint8_t], Default 0*/
     SAI_WRED_ATTR_WEIGHT = 0x0000000c,
```

Comment:
Hi
While the SAI model allows setting ECN and Wred enable per color,
If specific HW only support global control for ECN,
I can suggest implementation wise this logic :
If at least one color is enabled, enable global ECN, and if the user tries explicitly sets different values (for example, green ecn enable=true, yellow ecn enable=false), return not supported


From: Ashok Daparthi [mailto:notifications@github.com]
Sent: Wednesday, May 25, 2016 7:46 AM
To: opencomputeproject/SAI <SAI@noreply.github.com>
Cc: Itai Baz <itaib@mellanox.com>; Author <author@noreply.github.com>
Subject: Re: [opencomputeproject/SAI] allow enabling of ecn without wred in wred profile, change enable fro… (#112)


In inc/saiwred.h<https://github.com/opencomputeproject/SAI/pull/112#discussion_r64514518>:

> @@ -92,9 +106,6 @@ typedef enum _sai_wred_attr_t

>      /** 0 ~ 15 [sai_uint8_t], Default 0*/

>      SAI_WRED_ATTR_WEIGHT = 0x0000000c,

Hi Itai,

Please let me know, how to enable ECN in case ECN enable/disable is not supported by colour?

Regards
Ashok D

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/opencomputeproject/SAI/pull/112/files/350ea300b8bcad818c90c9326a2c1632bf2eb8ab#r64514518>